resources/js/components/Bard.js

```diff
@@ -33,18 +26,7 @@ class Bard {
     }
 
     get tiptap() {
-        return {
-            core,
-            commands,
-            utils,
-            extensions: {
-                ...extensions,
-                Link,
-                Image,
-                Subscript,
-                Superscript,
-            },
-         };
+        return { core };
```

Comment:
> @jacksleight & @jasonvarga we have added the core part back to the tiptap api: [c2abad0](https://github.com/statamic/cms/pull/6043/commits/c2abad052deac27978bf8c380e38b38383f61b59) We could still do that for the extensions and possibly @tiptap/vue-2 if that would make sense to you?

@jacksleight Just confirming... Are the extensions necessary? Is what @wiebkevogel done here good enough for you? You were the one that added extensions there in the first place. https://github.com/statamic/cms/pull/4314/files#diff-ea63b8582bcfd245a5180674f9116d89815b9f661a9df22947853a4ce76ea680R36-R47

I'm fine with just the `core` to be added, if that's enough for what most people would probably need in Tiptap 2.

Reply:
Yup fine for me, I no longer need access to the extension classes as TipTap2 can extend extensions in other ways.